src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -236,14 +252,16 @@ public static IServiceCollection AddApplicationInsightsTelemetryProcessor(this I
         /// <returns>
         /// The <see cref="IServiceCollection"/>.
         /// </returns>        
-        public static IServiceCollection ConfigureTelemetryModule<T>(this IServiceCollection services, Action<T> configModule) where T : ITelemetryModule
+        public static IServiceCollection ConfigureTelemetryModule<T>(this IServiceCollection services,
+            Action<T, ApplicationInsightsServiceOptions> configModule) where T : ITelemetryModule
```

Comment:
If we change the signature of this public method, existing customers will break upon upgrading. Can you avoid this breaking change?

Reply:
good point, I'll add an overload